langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -220,6 +308,11 @@ public isolated function toBytes(string str) returns byte[] = @java:Method {
 
 # Constructs a string from its UTF-8 representation in bytes.
 #
+# ```ballerina
+# string:fromBytes([72, 101, 108, 108, 111, 32, 66, 97, 108, 108, 101, 114, 105, 110, 97, 33]) ⇒ Hello, World!
+# string:fromBytes(base64 `lang`) ⇒ error
```

Comment:
Would this be confusing to someone who is not familiar with the byte array literal? How about
```suggestion
# string:fromBytes([149, 169, 224]) ⇒ error
```

Reply:
Makes sense. Will change.